### PR TITLE
Added GitHub Page Link of FLINT Handbook to Readme

### PR DESCRIPTION
**Pull request by meenaljain0303**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,11 @@ Here are the steps you can follow while setting up the documentation locally.
 
 If you choose to setup the documentation website locally, you need to have Python and the Python package manager `pip` on your local machine. Follow the official [Python installation](https://www.python.org/downloads/) to setup Python on your local machine.
 
-First make a fork, and then clone the repo:
+First make a fork, and then clone the repo
 
-2. Clone the repository. Replace the `<GITHUB_USERNAME>` with your GitHub username. You can find your username by clicking on your profile picture in the top right corner of the GitHub website.
+2. Clone the repository 
+
+Replace the `<GITHUB_USERNAME>` with your GitHub username. You can find your username by clicking on your profile picture in the top right corner of the GitHub website.
```
Comment (HarshCasper) [inline README.md:21]: Can you either put them in a single line, or indent it properly so that the text is part of the same block?

File: README.md [hunk 2]
```diff
@@ -14,9 +14,11 @@ Here are the steps you can follow while setting up the documentation locally.
 
 If you choose to setup the documentation website locally, you need to have Python and the Python package manager `pip` on your local machine. Follow the official [Python installation](https://www.python.org/downloads/) to setup Python on your local machine.
 
-First make a fork, and then clone the repo:
+First make a fork, and then clone the repo
```
Comment (HarshCasper) [inline README.md:17]: Why remove the colon?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Handbook
 
-This repository contains FLINT Handbook
+This repository contains [FLINT Handbook](https://moja-global.github.io/Handbook/)
```
Comment (HarshCasper) [inline README.md:3]: Can you rephrase this along these lines: https://github.com/moja-global/community-website#introduction

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (meenaljain0303) [reply README.md:3]: Yes surely

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@meenaljain0303 Please ensure that you sign your commit and that it adheres to the Developer Certificate of Origin (DCO) requirements before submitting it for review.

**Comment by HarshCasper** [comment]
Hi @meenaljain0303 — After making another commit, you can run these commands to sign off the DCO:

```sh
git rebase HEAD~2 --signoff
git push --force-with-lease origin main
```
